### ISSUE TITLE
Does fastify support the jsonp response method as express?

### ISSUE THREAD

**Issue by R1ZEN** [fastify/help#772]
## 💬  Question here 

Some libraries are based on the **jsonp** data acquisition format. Are there plugins for `fastify` that allow you to add this functionality?

Example:
```js
res.jsonp({ foo: 'bar' })
// => callback({ "foo": "bar" })
````
[express jsonp doc](https://expressjs.com/en/5x/api.html#res.jsonp)
[express jsonp implementation](https://github.com/expressjs/express/blob/master/lib/response.js#L293)

## Your Environment

- *node version*: 16.14.2
- *fastify version*: 3.0.0
- *os*: Mac

**Comment by Asjas** [comment]
@R1ZEN No, Fastify does not support `jsonp`. You will have to extend your Fastify server to add it as a response method.

You can search to see if any community Fastify plugins exist that already do that, but if you can't find any then you'll have to add it yourself

https://www.fastify.io/docs/latest/Reference/Plugins/#plugins

**Comment by R1ZEN** [comment]
Created a fastify plugin for `jsonp` in replay.: https://github.com/R1ZEN/fastify-jsonp
